applications/API/serializers.py

```
@@ -0,0 +1,487 @@
+# from rest_framework import serializers
```

Pylint (3.3.7)

[warning] Unknown option value for '--disable', expected a valid pylint message and got 'long-suffix'

(W0012)